### PR TITLE
Create - Signup Page

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Sign up page with 
- Create a new account
- Existing user
- Terms and Conditions

Temp-NouStore-Signup

### CODE REVIEW

**Review by atharvabhanage02 — APPROVED** [review]
Looks Good

File: css/signup.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/* Colors not decided yet, will use variables later */
+.login-wrapper {
+    background-color: aliceblue;
```
Comment (krishnakant01) [inline css/signup.css:3]: use variable here

File: css/signup.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    margin: auto;
+    margin-top: 25vh;
+    width: 20rem;
+    box-shadow: 0px 0px 10px 1px rgb(20, 113, 194);
```
Comment (krishnakant01) [inline css/signup.css:7]: try to avoid giving color to box shadow (suggested by vikram I think).

**Review by krishnakant01 — APPROVED** [review]
code looks good to merge, just look into some suggestions.

File: css/signup.css (hunk 1 above)
Comment (abhijitdotsharma) [reply css/signup.css:3]: Yes, thanks for pointing it out

File: css/signup.css (hunk 2 above)
Comment (abhijitdotsharma) [reply css/signup.css:7]: Will have to think of something else here. Thanks

**Pull request merged**